starcounter-include.html

```diff
@@ -63,7 +74,9 @@
                     },
                     viewModel: {
                         set: function(newValue) {
+                            this.settingPartialFromModel = true;
```

Comment:
If we will use `viewModel` internall (which we should in my opinion), that kind of magic will not be needed. Unless you do this just to reflect the old `partial` values set to `viewModel`